NickvisionMoney.GNOME/Views/NewAccountDialog.cs

```diff
@@ -72,6 +75,18 @@ private NewAccountDialog(Gtk.Builder builder, NewAccountDialogController control
                 ValidateName();
             }
         };
+        _accountPasswordRow.OnNotify += (sender, e) =>
+        {
+            if(e.Pspec.GetName() == "text")
+            {
+                ShowPasswordStrength();
+            }
+        };
+        _accountPasswordStrengthBar.SetMinValue(Convert.ToDouble((int)PasswordStrength.Blank));
+        _accountPasswordStrengthBar.SetMaxValue(Convert.ToDouble((int)PasswordStrength.VeryStrong));
+        _accountPasswordStrengthBar.AddOffsetValue("weak", 1);
+        _accountPasswordStrengthBar.AddOffsetValue("medium", 3);
+        _accountPasswordStrengthBar.AddOffsetValue("strong", 4);
```

Comment:
I don't quite understand, will the bar fill completely for very strong password?

Reply:
The problem of the bar not filling for very strong password is with `Nickvision.Aura.Credential.GetPasswordStrength()` for some reason it's not returning VeryStrong for anything...I need to take a look at it.

Reply:
Should all be good now:
![image](https://github.com/NickvisionApps/Denaro/assets/17648453/8ced8bc5-7913-470c-999d-6f8ee59cdbf4)